File: rest/ip-address/create-acl-address/create-acl-address.js
// Download the Node helper library from twilio.com/docs/node/install
// These vars are your accountSid and authToken from twilio.com/user/account
var accountSid = 'AC32a3c49700934481addd5ce1659f04d2';
var authToken = "{{ auth_token }}";
var client = require('twilio')(accountSid, authToken);

client.sip.ipAccessControlLists("AL32a3c49700934481addd5ce1659f04d2").ipAddresses.create({
    friendlyName: "My office IP Address",
    ipAddress: "55.102.123.124"
}, function(err, ipAddress) {
    process.stdout.write(ipAddress.sid);
});

File: rest/sip-in/create-acl-address/create-acl-address.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
class Example 
{
  static void Main(string[] args) 
  {
    // Find your Account Sid and Auth Token at twilio.com/user/account
    string AccountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    string AuthToken = "your_auth_token";
    var twilio = new TwilioRestClient(AccountSid, AuthToken);

    
    var ipAddress = twilio.CreateIpAddress("AL32a3c49700934481addd5ce1659f04d2", "My office IP Address", "55.102.123.124");
    
    Console.WriteLine(ipAddress.Sid);
  }
}